pkg/chunk/aws/dynamodb_storage_client.go

```diff
@@ -137,6 +138,7 @@ func (cfg *StorageConfig) RegisterFlags(f *flag.FlagSet) {
 	f.Var(&cfg.S3, "s3.url", "S3 endpoint URL with escaped Key and Secret encoded. "+
 		"If only region is specified as a host, proper endpoint will be deduced. Use inmemory:///<bucket-name> to use a mock in-memory implementation.")
 	f.BoolVar(&cfg.S3ForcePathStyle, "s3.force-path-style", false, "Set this to `true` to force the request to use path-style addressing.")
+	f.BoolVar(&cfg.S3ForceTLS, "s3.force-tls", false, "Set this to `true` to force s3 requests to be made over https")
```

Comment:
Nit: `Set this to true to.." is a bit redundant, could just say "Force S3e requests to be made over HTTPS." 

If you change it, please change the one above to.

Reply:
Fair, was trying to be consisted with what was above :)